release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -456,7 +466,7 @@ For more information, see xref:../updating/preparing_for_updates/updating-cluste
 === Networking
 
 [id="ocp-4-19-networking-support-load-secrets_{context}"]
-==== Creating a route with externally managed certificate (General Availability) 
+==== Creating a route with externally managed certificate (General Availability)
```

Comment:
```suggestion
==== Creating a route with externally managed certificates (General Availability)
```
Should "certificates" be plural here? Or should the wording be "Creating a route with an externally managed certificate"?